python/mp/effects/set_color.py

```diff
@@ -0,0 +1,20 @@
+
+from mp.effects import effect
+from mp import color
+
+class SetColor(effect.Effect):
+    """
+    SetColor is a one-shot effect that sets all beads in a set to a given color.
+    It removes itself from the mainloop after one invocation of next()
+    """
+
+    def __init__(self, bead_set, color=color.Color()):
+        super().__init__("set_color", bead_set, color=color)
+
+    def next(self, rosary):
+        super().next()
+
+        for bead in (self.bead_list):
+            bead.copy_color(self.color)
+
+        self.finished = True
```

Comment:
Aha, I'd wondered what SetColor was. Thought maybe it was like "a set of colors" and figured since I didn't obviously see it being used, whatevs.

Should've been more diligent!